### DISCUSSION TITLE
Form will not reset after submit

### DISCUSSION THREAD

**Discussion by mihaimazareanu** [neuefische/questions#363] [Web] [answered]
Hi,

I have built a handleSubmitForm function that works wonderfully, it creates a new user in my DB on MongDB, but although I have written `event.target.reset()` in my code, the input fields in the form will not reset. 

Any idea why?

Here is a PR, to see the whole code, if anyone needs it.

https://github.com/mihaimazareanu/capstone-project-kidsfi/pull/14

**Comment by actyralf** [comment, marked as answer]
You are using controlled inputs (i.e. you assign state values to your inputs) so you have to reset your state to it's initial values to reset the form.

**Comment by mihaimazareanu** [comment]
Thanks for the explanation, I used setInput() to the initial state (empty object) and it's working now. 🙏
